lib/impress.constants.js

```diff
@@ -172,9 +173,8 @@ impress.DEFAULT_API = [
 
 // Preparing stack trace transformations
 impress.STACK_REGEXP = [
-  [api.path.dirname(__dirname) + api.common.pathSeparator + 'node_modules', ''],
-  [__dirname + api.common.pathSeparator, ''],
   [process.cwd() + api.common.pathSeparator + 'node_modules', ''],
```

Comment:
2.3. `api.common.pathSeparator` is clearly a constant from the logical point of view, even if it's assigned conditionally, so it should be uppercase.